src/wfs/src/main/java/org/geoserver/wfs/response/CSVOutputFormat.java

```diff
@@ -88,7 +90,6 @@ protected void write(
             FeatureCollectionResponse featureCollection, OutputStream output, Operation getFeature)
             throws IOException, ServiceException {
         // write out content here
-
```

Comment:
The whitespace changes probably are not needed and could be worth cleaning up if you are making other changes. Otherwise noise.